.github/workflows/osx.yml

```diff
@@ -0,0 +1,31 @@
+name: osx build
+
+on:
+  push:
+    branches:
+    - develop
+    - main
+    - feature/macos-build
```

Comment:
Need to remove. After PR is merged to develop, pipeline can be run for any branch from workflow_dispatch trigger.